man/openqa-label-all.1.in

```diff
@@ -0,0 +1,65 @@
+.\"                                      Hey, EMACS: -*- nroff -*-
```

Comment:
Is this file generated?

Reply:
it's based on this [template](https://salsa.debian.org/debian/dh-make/-/blob/master/lib/debian/manpage.1.ex) from the dh_make packaging tool, and then edited by hand.